api/src/repositories/submission-repository.test.ts

```diff
@@ -174,6 +175,93 @@ describe('SubmissionRepository', () => {
     });
   });
 
+  describe('updateSubmissionRecordEMLSource', () => {
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    it('should throw an error when update sql fails', async () => {
+      const mockQueryResponse = { rowCount: 0 } as any as Promise<QueryResult<any>>;
+
+      const mockDBConnection = getMockDBConnection({
+        sql: async () => {
+          return mockQueryResponse;
+        }
```

Comment:
(dont actually need to change anything) If you wanted to reduce lines of code, you could write this as:

```
sql: () => mockQueryResponse
```